webpack/ForemanTasks/Components/TasksTable/TasksTableHelpers.js

```diff
@@ -0,0 +1,12 @@
+import URI from 'urijs';
+
+export const updateURlQuery = query => {
+  const uri = new URI(window.location.href);
+  uri.setSearch(query);
+  window.history.pushState({ path: uri.toString() }, '', uri.toString());
```

Comment:
Shouldn't the `history` object suppose to get imported from `foremanReact`?

Reply:
It doesn't change the browsers URL(only changes the path in the the `ReactApp`)

Reply:
The history in foremanReact only changed the path in the main Router component and not in the browser it self. In the next next PR the history object is from the router